docs/src/Guides/03 Creating Commands.md

```diff
@@ -284,6 +296,38 @@ async def autocomplete(self, ctx: AutocompleteContext):
     )
 ```
 
+## An example `/kick` command
+
+Let's use some of the above features to create a real-world command that kicks a member out of the guild. Remember to add this code to the [Basic bot](/interactions.py/Guides/01 Getting Started/#basic-bot).
+
+```python
+from interactions import SlashContext, Permissions, Embed, Color
+from interactions import slash_command, slash_option
+
+@slash_command(name='kick', description='Kicks a user')
```

Comment:
Why isn't this command disabled for DMs or enabled only for users with kick permissions? Both of those are easy to set up here.